CONTRIBUTORS.md

```diff
@@ -1,51 +1,51 @@
 # PursuedPyBear Contributors
 
-
 ## Maintainers
 
-Name | Email | Twitter |
------|-------|---------|
-[Piper Thunstrom](https://github.com/pathunstrom) | [pathunstrom@gmail.com](mailto:pathunstrom@gmail.com) | [pathunstrom](https://twitter.com/pathunstrom)
-[Luna Lunapiena](https://github.com/lunacodes) | [lunacodes@gmail.com](lunacodes@gmail.com) | [lunacodess](https://twitter.com/lunacodess)
-[Jamie Bliss](https://github.com/AstraLuma/) | [jamie@ivyleav.es](jamie@ivyleav.es) | [AstraLuma](https://twitter.com/AstraLuma)
+| Name                                              | Email                                                 | Twitter                                        |
```

Comment:
This is a lot of formatting changes, can we revert this file and just add your information?

Trying to keep the nice text format will be a losing game over time regardless of how large we make it.

Reply:
I realised that one of my extensions in vscode was causing the formatting upon saving. I disabled it and will commit again. So the nice format is back :)